scripts/process_vcs.py

```diff
@@ -314,8 +313,10 @@ def vcs_download(obsid, start_time, stop_time, increment, head, data_dir,
             #        submit_slurm(tar_batch,body,batch_dir=working_dir+"/batch/", slurm_kwargs={"time":"1:00:00", "partition":"gpuq" })
 
 
-
-            module_list=["mwa-voltage/master"]
+            
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4822105)